cernopendata/modules/fixtures/cli.py

```diff
@@ -246,7 +248,10 @@ def datasets():
                 files = data.pop('files', None)
 
                 id = uuid.uuid4()
-                cernopendata_recid_minter(id, data)
+                if data['doi']:
```

Comment:
What about `data.get('doi')` in case the key may not be there? E.g. this could be important in output templates... tying to use the same coding style how to access record fields.